test/test_scenario_perception_pipeline.cpp

```diff
@@ -7,92 +7,165 @@
 #include <moveit/moveit_cpp/planning_component.h>
 
 #include <geometry_msgs/msg/pose.hpp>
+#include <nav_msgs/msg/path.hpp>
+#include <dynmsg/msg_parser.hpp>
+#include <dynmsg/typesupport.hpp>
+#include <dynmsg/yaml_utils.hpp>
+
+#include <ament_index_cpp/get_package_share_directory.hpp>
+#include <iostream>
 
 using moveit::planning_interface::MoveGroupInterface;
 
-// class MoveGroupServer {
-// };
+const std::string PLANNING_GROUP = "panda_arm";
 
-class ScenarioPerceptionPipeline {
+class ScenarioPerceptionPipeline
```

Comment:
In the header file you should add documentation for every class and (member) function you create to make it easier to understand